models/lorenz_63/README.rst

```diff
@@ -0,0 +1,134 @@
+################
+Lorenz 63 README
+################
+
+Contents
+========
+
+#. `Overview`_
+#. `Namelist`_
+#. `Terms of Use`_
+#. `References`_
+
+Overview
+========
+
+This 3-variable model was described in Lorenz (1963). [1]_ In Lorenz 63, DART
+advances the model, gets the model state and metadata describing this state,
+finds state variables that are close to a given location, and does spatial
+interpolation for model state variables. The distinctive part of the model
+interface is the `namelist`_.
```

Comment:
this model has a workshop_setup.csh   and is extensively used in the DART tutorials. That should be mentioned.

Reply:
Since the stated purpose of this pull request is to convert the documentation from HTML to reStructuredText, I think it makes sense to add the workshop_setup.csh descriptions in a different pull request.